controllers/enforcer.go

```diff
@@ -79,7 +91,13 @@ func (c *ApiController) Enforce() {
 
 	listPermissionIdMap := object.GroupPermissionsByModelAdapter(permissions)
 	for _, permissionIds := range listPermissionIdMap {
-		enforceResult, err := object.Enforce(permissionIds[0], &request, permissionIds...)
+		firstPermission, err := object.GetPermission(permissionIds[0])
```

Comment:
firstPermission is nil?

Reply:
it cannot be nil because we get the `permissionIds` it from existed permissions